### PR TITLE
Ftr/11266

### PR DESCRIPTION
**Pull request by JimLiuBAH**
**High level description:**

Rewording DATA Act Broker in usaspending website and documentation

**Technical details:**

**JIRA Ticket:**
[DEV-11266](https://federal-spending-transparency.atlassian.net/browse/DEV-11266)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Comment by brianpetway** [comment]
Looks like there are two more that need to change.

In src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx, line 292, need to rename “DATA Act Broker Submission” to “Data Accountability Broker Submission”.

And there's one instance of 'DATA Act Broker', in src/content/about-the-data/data-validation-of-source-systems.mdx, line 1.

**Comment by JimLiuBAH** [comment]
> Looks like there are two more that need to change.
> 
> In src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx, line 292, need to rename “DATA Act Broker Submission” to “Data Accountability Broker Submission”.
> 
> And there's one instance of 'DATA Act Broker', in src/content/about-the-data/data-validation-of-source-systems.mdx, line 1.

Looks like my VSCode search was omitting some directories. Replaced now

File: src/content/about-the-data/file-a.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 File A (Account Balances) is part of the package of data submitted
 to USAspending.gov every month by federal agencies via the DATA Act
```
Comment (brianpetway) [inline src/content/about-the-data/file-a.md:2]: here's another one; 'Data Act Broker' -> 'Data Broker'

File: src/content/about-the-data/data-validation-of-source-systems.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-The DATA Act Broker validates the data it receives from agency financial systems
+The Data Accountability Broker validates the data it receives from agency financial systems
```
Comment (brianpetway) [inline src/content/about-the-data/data-validation-of-source-systems.mdx:1]: this one should've changed from 'Data Act Broker' to 'Data Broker'

**Comment by brianpetway** [comment]
Two more small changes; inline comments

**Review by brianpetway — APPROVED** [review]

**Pull request merged**